lib/internal/errors.js

```diff
@@ -124,6 +140,7 @@ E('ERR_HTTP_INVALID_STATUS_CODE',
 E('ERR_INDEX_OUT_OF_RANGE', 'Index out of range');
 E('ERR_INVALID_ARG_TYPE', invalidArgType);
 E('ERR_INVALID_CALLBACK', 'callback must be a function');
+E('ERR_INVALID_ENCODING', 'The "encoding" argument must be a valid encoding');
```

Comment:
See my comment below. This type is at least redundant to `ERR_UNKNOWN_ENCODING` but I would use `ERR_INVALID_OPT_VALUE` instead.